handlers/response/writer_test.go

```diff
@@ -0,0 +1,75 @@
+package response
+
+import (
+	. "github.com/smartystreets/goconvey/convey"
+	"testing"
+	"net/http/httptest"
+	"net/http"
+	"encoding/json"
+)
+
+type child struct {
+	Name string `json:"value"`
+}
+
+type parent struct {
+	Name  string `json:"name"`
+	Child child `json:"child"`
+}
+
+type mockOnsJsonEncoder struct {
+	invocationArgs  []interface{}
+	mockedBehaviour func(w http.ResponseWriter, value interface{}, status int) error
+}
+
+func mockEncoderInit() *mockOnsJsonEncoder {
```

Comment:
go lint is saying that 'mockOnsJsonEncoder' should be mockOnsJSONEncoder. Should we even upper case 'Ons'?